static/app/components/performance/waterfall/utils.tsx

```diff
@@ -174,7 +174,7 @@ type Rect = {
 };
 
 // get position of element relative to top/left of document
-const getOffsetOfElement = (element: Element) => {
+export const getOffsetOfElement = (element: Element) => {
```

Comment:
I will follow up with a pull request to move these functions into `static/app/utils/`.